ttnn/cpp/ttnn/device_operation.hpp

```diff
@@ -489,7 +503,7 @@ typename device_operation_t::tensor_return_value_t run(
 
     using tensor_return_value_t = typename device_operation_t::tensor_return_value_t;
     static_assert(not std::same_as<tensor_return_value_t, void>, "Operation return type cannot be \"void\"");
-
+    // ADD GraphTracker::instance().track_device_operation
```

Comment:
Is this a leftover comment?

Reply:
We are in the middle of discussion with Akhmed on it. He want to see device op as the leaf node.
But maybe it will be separate feature.